creation/lib/cgWDictFile.py

```diff
@@ -286,6 +353,12 @@ def load_common_dicts(dicts, description_el):  # update in place
 
 
 def load_main_dicts(main_dicts):  # update in place
+    """Load (from file) the entry dictionaries
```

Comment:
Should “entry” be changed to “main” here?

Reply:
done